### PR TITLE
gui-apps/waypipe: fix emesonargs and move to CPU_FLAGS_ARM

### PR DESCRIPTION
**Pull request by epsilon-0**
Package-Manager: Portage-3.0.16, Repoman-3.0.2
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

**Comment by epsilon-0** [comment]
@Freed0mBear another ping

**Comment by epsilon-0** [comment]
@Freed0mBear another ping :D

File: gui-apps/waypipe/waypipe-0.8.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+LICENSE="MIT"
+SLOT="0"
+
+WAYPIPE_FLAG_MAP_X86=( avx2:with_avx2 avx512f:with_avx512f sse3:with_sse3 )
+WAYPIPE_FLAG_MAP_ARM=( neon:with_neon_opts )
+WAYPIPE_FLAG_MAP=(
+	"${WAYPIPE_FLAG_MAP_X86[@]/#/cpu_flags_x86_}"
+	"${WAYPIPE_FLAG_MAP_ARM[@]/#/cpu_flags_arm_}"
+)
```
Comment (juippis) [inline gui-apps/waypipe/waypipe-0.8.0.ebuild:30]: This just looks wrong to me, but my brain doesn't bend to this anymore and seems to build fine for me.

**Review by juippis — APPROVED** [review]

File: gui-apps/waypipe/waypipe-0.8.0.ebuild (hunk 1 above)
Comment (epsilon-0) [reply gui-apps/waypipe/waypipe-0.8.0.ebuild:30]: LOLOLOLOL

**Pull request closed without merging**